packages/Testing/PHPUnit/Behavior/MultipleFilesChangedTrait.php

```diff
@@ -85,7 +85,7 @@ private function checkAdditionalChanges(array $expectedFileChanges): void
         foreach ($addedFilesWithContent as $addedFileWithContent) {
             [, $addedFilePathWithContentFilePath] = explode(
                 '_temp_fixture_easy_testing',
-                $addedFileWithContent->getFilePath()
+                (string) $addedFileWithContent->getFilePath()
```

Comment:
it seems got `PHPStan\Type\MixedType`

```bash
PHPStan\Type\MixedType #60872
   subtractedType: null
   isExplicitMixed: false
```

Reply:
resolved by skip method call on trait as its property of method call can be from Class_: https://github.com/rectorphp/rector-src/pull/1655/commits/be98cf0d5f9f41ae204d509ff9e1844beca5fb89